AUTHORS

```diff
@@ -0,0 +1,6 @@
+# This is the list of FirstOrderLp authors for copyright purposes.
+#
+# This does not necessarily list everyone who has contributed code, since in
+# some cases, their employer may be the copyright holder.  To see the full list
+# of contributors, see the revision history in source control.
+Google LLC
```

Comment:
I believe we should also list Mateo Diaz - he has contributed code under a non-Googler CLA, so isn't coverged by Google LLC for copyright purposes.

Reply:
Done